src/core/InformationPanel/InformationPanelContent.tsx

```diff
@@ -0,0 +1,76 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+import cx from 'classnames';
+import React from 'react';
+import { CommonProps, useTheme } from '../utils';
+import { useContainerWidth } from '../utils/hooks/useContainerWidth';
+import '@itwin/itwinui-css/css/information-panel.css';
+
+export type InformationPanelContentProps = {
+  /**
+   * Maximum width of the panel below which layout will shift from two columns to one,
+   * i.e. label and input will display on separate lines for widths lower than this value.
+   *
+   * If set to 0, the layout will always be two columns.
+   * If set to `Number.POSITIVE_INFINITY`, the layout will always be one column.
+   *
+   * @default 0
+   */
+  responsiveBreakpoint?: number;
+  /**
+   * Content of the component.
+   * Should be pairs of `Label` and input components.
+   */
+  children: React.ReactNode;
+} & Omit<CommonProps, 'title'>;
+
+/**
+ * The `InformationPanelContent` component should be used inside `InformationPanelBody`
```

Comment:
Too long description, no one will read, sorry 😀

Reply:
Cant hurt to have. All the info is useful to understand behavior